@here/harp-mapview/lib/ThemeLoader.ts

```diff
@@ -38,7 +38,13 @@ export class ThemeLoader {
         return this.resolveUrls(theme);
     }
 
-    private static resolveUrls(theme: Theme): Theme {
+    /**
+     * Resolves all [[Theme]]'s relatives URLs to full URL using the document's base URL
```

Comment:
It resolves relative URLs versus `theme.url` not document's base URL.

Reply:
True, will change.